test/api_test.js

```diff
@@ -58,7 +58,7 @@ describe("Api", () => {
     describe("full URL", () => {
       var endpoints;
 
-      beforeEach(() => endpoints = api.endpoints({fullUrl: true}))
+      beforeEach(() => endpoints = api.endpoints({ fullUrl: true }))
```

Comment:
Yes, I could not resist to change the few ones I saw... But there may be a way to configure jslint